docs/get-started/json-rpc-commands.mdx

```diff
@@ -0,0 +1,879 @@
+````
+
+### eth_getBlockTransactionCountByNumber
+
+Returns the number of transactions in a block matching the given block number.
+
+---
+
+<h4><i>Parameters:</i></h4>
+
+*  <b>  QUANTITY|TAG </b> - integer of a block number, or the string "latest"
+
+<h4><i>Returns:</i></h4>
+
+*  <b>  QUANTITY </b> - integer of the number of transactions in this block.
+
+<h4><i>Example:</i></h4>
+
+Run command and see live results from our testnet.
+
+````bash
+curl  https://rpc.poa.psdk.io:8545 -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_getBlockTransactionCountByNumber","params":["latest"],"id":1}'
+````
+
+<JsonRpcTerminal method="eth_getBlockTransactionCountByNumber" params={[]} network="https://rpc.poa.psdk.io:8545"/>
+
+### eth_getLogs
+
+Returns an array of all logs matching a given filter object.
+
+---
+
+<h4><i>Parameters:</i></h4>
+<b> Object </b>  - The filter options:
+
+*  <b> fromBlock: QUANTITY|TAG </b> - (optional, default: "latest") Integer block number, or "latest" for the last mined block 
+*  <b> toBlock: QUANTITY|TAG </b> - (optional, default: "latest") Integer block number, or "latest" for the last mined block 
+*  <b> ddress: DATA|Array, 20 Bytes </b> - (optional) Contract address or a list of addresses from which logs should originate.
+*  <b> topics: Array of DATA </b> - (optional) Array of 32 Bytes DATA topics. Topics are order-dependent. Each topic can also be an array of DATA with “or” options.
+*  <b> blockhash: DATA, 32 Bytes </b> - (optional, future) With the addition of EIP-234, blockHash will be a new filter option which restricts the logs returned to the single block with the 32-byte hash blockHash. Using blockHash is equivalent to fromBlock = toBlock = the block number with hash blockHash. If blockHash is present in in the filter criteria, then neither fromBlock nor toBlock are allowed.
+
+<h4><i>Returns:</i></h4>
+
+*  <b> QUANTITY </b> - integer of the number of transactions send from this address.
+
+<h4><i>Example:</i></h4>
+
+````bash
+curl  https://rpc.poa.psdk.io:8545 -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_getLogs","params":[{"topics": ["0x000000000000000000000000a94f5374fce5edbc8e2a8697c15331677e6ebf0b"]}],"id":1}'
+````
+
+
+### eth_getCode
+
+Returns code at a given address.
+
+---
+
+<h4><i>Parameters:</i></h4>
+
+*  <b>  DATA, 20 Bytes </b> - address
+*  <b>  QUANTITY|TAG </b> - integer block number, or the string "latest"
+
+<h4><i>Returns:</i></h4>
+
+*  <b>  DATA </b> - the code from the given address.
+
+<h4><i>Example:</i></h4>
+
+````bash
+curl  https://rpc.poa.psdk.io:8545 -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_getCode","params":["0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b", "0x2"],"id":1}'
+````
+
+### eth_call
+
+Executes a new message call immediately without creating a transaction on the block chain.
```

Comment:
"block chain" should be 1 word

Reply:
Fixed! 
 [b1173b4](https://github.com/0xPolygon/polygon-sdk-docs/pull/41/commits/b1173b4db2a92b5f3c153e1906ad8ac9e82900bf)